### PR TITLE
Update scenarios related to container images

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/integration-tests/issues/225

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: features/steps/container_images.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                "but got {entry['thoth_s2i']} instead"
+            )
+            break
+    else:
+        assert (
+            False
+        ), f"Thoth container image {container_image!r} not available in the Thoth container images listing on User API"
+
+
+@then("I should see {count} Thoth container images available")
```
Comment (mayaCostantini) [inline features/steps/container_images.py:61]:
```suggestion
@then("I should see "{count}" Thoth container images available")
```

**Review by mayaCostantini — COMMENTED** [review]
Just two small doubts but otherwise lgtm :+1: :100:

File: features/steps/container_images.py (hunk 1 above)
Comment (fridex) [reply features/steps/container_images.py:61]: This one should not be needed (parameters do not need to be quoted).

Comment (mayaCostantini) [reply features/steps/container_images.py:61]: I was not sure if quotes were needed or not at they were present at L40 and also in #229

**Review by mayaCostantini — APPROVED** [review]

**Pull request merged**